templates/index.html

```diff
@@ -275,11 +274,11 @@ <h3 class="sponsors__amount">$100 / month</h3>
                 </a>
             </div>
         </div>
-            <h2 class="past-sponsors-title">Past Sponsors</h2>
-            <div>
-                Futurewei, Legion Labs, Striked, Metabuild 
-            </div>
+        <h2 class="past-sponsors-title">Past Sponsors</h2>
+        <div>
+            Futurewei, Legion Labs, Striked, Metabuild
         </div>
     </div>
 </div>
+</div>
```

Comment:
Indentation fix above is good, but I'm pretty sure this is an extra `</div>`.

Reply:
oops, I think my auto-format extension got a bit excited. I'll take a closer look at the diff and try to minimize change.

Reply:
Well this is spooky. When I compare my branch locally, this diff does not show up, only the foresight section is changed.

Reply:
So, the code looks different in the deff viewer, but the indent is not highlighted... at all. That is concerning. Not sure what is going on there, but I removed the extra `div`.

Reply:
Also not sure what's going on here but in the current changeset the open/close tags seem balanced.